web-client/views/upload.jade

```diff
@@ -382,6 +391,43 @@ html
                   input(id='dashedGrayLineButton', type='checkbox', value='dashedGrayLine')
                   label(for='dashedGrayLineButton' class='sideLabel') Show Gray Edges as Dashed
 
+        //- Need to make this work.
+        div(class="data-sidebar disabled panel panel-default")
+          p(class='panel-title sidebarPanelHeader disabledOnLaunch')
+            a(id='data-sidebar' data-toggle='collapse' data-target='#dataSidebarPanel') Data
+          div(id="dataSidebarPanel" class="panel-collapse collapse")
+            div(class="panel-body")
+              div(class="dataSidebarBody")
+                p(class='sideHeader info' data-toggle='tooltip') Dataset
+                select(multiple id="data-set-select")
+                    //- Checkboxes instead of ctrl click ?
+                    //- Eventually want it to be so that the possible available ones can be detected from the network sheet
+                    //- Refresh selection upon new upload
+                    //- Add "onclick" functionality for checkbox
+                    option(value="data-set-select-barreto") Barreto 2018 wt
+                    option(value="data-set-select-dahlquistdcin5") Dahlquist 2018 dcin5
+                    option(value="data-set-select-dahlquistdgln3") Dahlquist 2018 dgln3
+                    option(value="data-set-select-dahlquistdhap4") Dahlquist 2018 dhap4
+                    option(value="data-set-select-dahlquistdzap1") Dahlquist 2018 dzap1
+                    option(value="data-set-select-dahlquistwt") Dahlquist 2018 wt
+                    option(value="data-set-select-kitagawa") Kitagawa 2002 wt
+                    option(value="data-set-select-thorsen") Thorsen 2007 wt
```

Comment:
Next step: This part should eventually become dynamic. Instead of being hardcoded in _upload.jade_, this should start out empty, and a web service call to the server should exist which returns this list.